Makefile

```diff
@@ -472,6 +472,7 @@ release-binary: $(RELEASE_DIR)
 
 .PHONY: release-staging
 release-staging: ## Builds and push container images to the staging bucket.
+	docker pull docker.io/docker/dockerfile:1.0-experimental
```

Comment:
Should this tag just be `experimental` now that the annotation in the dockerfile was updated to not use `1.0-experimental`?

Reply:
yep. it is available